SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 #include "SpatialGDKEditorSettings.h"
 #include "Settings/LevelEditorPlaySettings.h"
+#include "MessageDialog.h"
```

Comment:
Can you fix these includes to be alphabetical?

Reply:
I recommend you guys use the .clang-format, no need to care about code formatting.